contracts/ExitToken.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity ^0.4.24;
+
+import "openzeppelin-solidity/contracts/token/ERC721/ERC721BasicToken.sol";
+import "./ParsecBridge.sol";
+import "./TxLib.sol";
+import "./TransferrableToken.sol";
+
+contract ExitToken is ERC721BasicToken {
+
+  // event Debug(bytes data);
+  event ProxyExit(address exiter, uint256 utxoId);
+
+  ParsecBridge public bridge;
+
+  constructor(address b) public {
+    bridge = ParsecBridge(b);
+  }
+
+  function proxyExit(bytes32[] _proof, uint256 _oindex) public {
+    (TransferrableToken native, ) = bridge.tokens(0);
+    uint256 exitStake = bridge.exitStake();
+    native.transferFrom(msg.sender, this, exitStake);
+    ERC20(native).approve(bridge, exitStake);
+    uint256 utxoId = uint256(bridge.startExit(_proof, _oindex));
+    address exiter = recoverSigner(_proof);
+    _mint(exiter, utxoId);
+    emit ProxyExit(exiter, utxoId);
```

Comment:
would be good to remove exit from `bridge.exits` storage once the exit is paid out. Otherwise this list will grow forever bloating the blockchain

Like it was in a previous version of `finalizeExits`: https://github.com/parsec-labs/parsec-contracts/pull/60/files#diff-d0656b371926e0ba0475cc731f1be86eL531

Reply:
The exit data can't be deleted before the token is burned, since it contains the data needed for payouts (amount, stake and whether is it finalized). We could delete it after the NFT is burned though. But in that case the Bridge has to be aware of the NFT contract again, since only the NFT should be able to delete it. 

The more I think about this the more coupled the ExitToken and Bridge seem to be in nature...

Reply:
Or maybe we could just keep that data in the ExitToken contract actually.

Reply:
ah, sorry.. I put the code pointer on the wrong line. I suggested to remove exit after the `_burn` call, yes.

> But in that case the Bridge has to be aware of the NFT contract again

is this a problem? I would naturally try to do it the other way: make bridge to know about ExitToken (e.g. pass it's address in the constructor) and ExitToken not to be bridge-aware.

Reply:
> is this a problem? I would naturally try to do it the other way: make bridge to know about ExitToken (e.g. pass it's address in the constructor) and ExitToken not to be bridge-aware.

Not really, could do that also. It is a design choice and I am agnostic here, so I could change it. And sorry for late response, I haven't seen this comment until now.